### ISSUE TITLE
Add option to pass additional arguments to the startup command.

### ISSUE THREAD

**Issue by ConorIA** [Teamwork/node-auto-launch#46]
It would be great if there was an additional option to add `--` prefixed arguments to the start-up file (at least the Linux .desktop file). As an example, I use Wire Desktopm, which can be launched minimized by passing the `--startup` flag to the `wire` command, however there is no option to add `--startup` to the desktop file generated by auto-launch. Thanks!

**Comment by adam-lynch** [comment]
I'm open to it but I guess it should be supported for all platforms. Pull-requests are welcome 😄

**Comment by ConorIA** [comment]
I actually took a look at the code before submitting, but was a little intimidated (I'm not very well versed in Javascript). Anyway, I'll tinker this week and see what I come up with. If someone else happens along this issue and wants to take a stab at it, have at!

**Comment by erguotou520** [comment]
+1

**Comment by damianobarbati** [comment]
Yeah indeed useful! I need to detect whether the app was autostarted or not.

**Comment by MattiasFestin** [comment]
I also need this to detect if my app was autostarted or user started. So I can prevent showing a loading splashscreen on boot in a electron app.
